### DISCUSSION TITLE
Encyclopedia?

### DISCUSSION THREAD

**Discussion by Ohrldu** [ReadySetGooo/NoNameYet#11] [Ideas]
It would be cool for all the information, including externally linked websites, on this site to be converted into an encyclopedia. I know there's ways to covert web pages to PDFs. I wonder if there's an easy way to convert entire sites with many pages into pdfs since I know some of these are large archives. And maybe those pdfs can be converted into encyclopedias that can be downloaded or even printed.

**Comment by ReadySetGooo** [comment]
I was thinking of making pdfs for the purpose of using it to train an ai chat, kinda like chat gpt for the sdgs. It would be great to have the information in a printed format, like pdfs. Maybe save space by making the font super tiny and use a magnifying glass to read it. It'd be valuable for the sake of archiving
